openrtx/src/core/state.c

```diff
@@ -80,6 +80,12 @@ void state_init()
     state.emergency     = false;
     state.txDisable     = false;
     state.step_index    = 4; // Default frequency step 12.5kHz
+    #ifdef CONFIG_SPECTRUM
+    state.spectrum_startFreq = 432100000; // Default start frequency for spectrum display
```

Comment:
The initialization value should be dependent on the hardware configuration, otherwise on VHF radios it won't work. See for example https://github.com/OpenRTX/OpenRTX/blob/master/openrtx/src/core/cps.c#L46